src/Cli/Commands/InitOptions.cs

```diff
@@ -103,6 +105,9 @@ public class InitOptions : Options
         [Option("graphql.disabled", Default = false, Required = false, HelpText = "Disables GraphQL endpoint for all entities.")]
         public bool GraphQLDisabled { get; }
 
+        [Option("rest.request-body-not-strict", Default = false, Required = false, HelpText = "When set to true, allows extraneous fields in the request body for REST.")]
+        public bool RestRequestBodyNotStrict { get; }
```

Comment:
> Can we have the RFC's property name used here too? request-body-strict defaults to true, and if you want otherwise, you have to supply the value false.

No, we cannot.

> Can we have the RFC's property name used here too? request-body-strict defaults to true, and if you want otherwise, you have to supply the value false.

I was thinking about this as well earlier but relaxed/flexible request body didn't kind of make straight sense to me. I can give these suggestions to Jerry and then rename the property accordingly.

Reply:
Renamed to `request-body-flexible`